ext/lists.go

```diff
@@ -159,6 +187,29 @@ func (lib listsLib) CompileOptions() []cel.EnvOption {
 			),
 		)
 	}
+	if lib.version >= 2 {
+		sortDecl := cel.Function("sort",
+			templatedOverloads(comparableTypes, func(t *cel.Type) cel.FunctionOpt {
+				return cel.MemberOverload(
+					fmt.Sprintf("list_%s_sort", t.TypeName()),
+					[]*cel.Type{cel.ListType(t)}, cel.ListType(t),
+					cel.UnaryBinding(func(arg ref.Val) ref.Val {
```

Comment:
I think it's possible to introduce this function as a `cel.SingletonUnaryBinding` since the implementation is not type-specific. This allows people to add new signatures for comparable types without having to generate new functions to take advantage of `sort()`

Reply:
Got it. Can you check now? I added all list traits since adding only the necessary ones (`indexer` and `sizer`) would also match Map